### PR TITLE
Migrated GSOC 19 page to layer5-ng

### PR DESCRIPTION
**Pull request by ashutosh1401**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

I have successfully migrated GSOC 19 page to layer5-ng and page is working well and responsive in all devices.

### CODE REVIEW

**Comment by ashutosh1401** [comment]
#908 issue solved

File: src/collections/programs/gsoc-2019/index.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    </ul>
+    </div>
+    <div>
+    <h3>Project Ideas</h3>
+        <ul>
+            <b>Meshery</b>
+            <p>Meshery is the service mesh management plane for lifecycle, configuration and performance management of many service meshes. See <a href="https://layer5.io/meshery">Meshery</a> for more information.</p>
+            <b>Linkerd and Envoy</b>
+            <p>Linkerd is an ultralight service mesh for Kubernetes and beyond: <a href="https://linkerd.io">https://linkerd.io</a> . Envoy is an open source edge and service proxy, designed for cloud-native applications: <a href="https://www.envoyproxy.io">https://www.envoyproxy.io</a></p>.
+            <b>Benchmarks for Linkerd and Envoy</b>
```
Comment (Jashpatel1) [inline src/collections/programs/gsoc-2019/index.mdx:48]: A typo in the previous line. The full stop gets out of the paragraph. Rest looks good.

Comment (ashutosh1401) [reply src/collections/programs/gsoc-2019/index.mdx:48]: Now placed full stop correctly

**Review by Jashpatel1 — DISMISSED** [review]
Nice work here. Thanks @ashutosh1401

**Comment by Nikhil-Ladha** [comment]
I know the design has been kept the same as in the current site, but I would request you to change it to the design similar to the GSoC 2020 page on the new site.
https://layer5ng.netlify.app/programs/gsoc/2020

**Comment by ashutosh1401** [comment]
@Nikhil-Ladha  so i need to add gsoc logo , add bullet as same in gsoc 2020 page and place iframe to the bottom

**Comment by Tanuj22** [comment]
> @Nikhil-Ladha so i need to add gsoc logo , add bullet as same in gsoc 2020 page and place iframe to the bottom

Yes @ashutosh1401

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha so i need to add gsoc logo , add bullet as same in gsoc 2020 page and place iframe to the bottom

Yeah, just make the design similar don't add anything extra except the GSoC logo.

File: src/collections/programs/gsoc-2019/index.mdx [hunk 2]
```diff
@@ -10,11 +10,33 @@ import Button from "../../../reusecore/Button";
 import gsoc_logo from "./gsoc.svg";
 
 <ProgramsWrapper>
-<h2>Google Summer of Code 2019 and Layer5 Community</h2>
+<img src={gsoc_logo}/>
+<b>Google Summer of Code 2019 and Layer5 Community</b>
```
Comment (Nikhil-Ladha) [inline src/collections/programs/gsoc-2019/index.mdx:14]: Layer5 Community and GSoC

Comment (ashutosh1401) [reply src/collections/programs/gsoc-2019/index.mdx:14]: Any further changes required

Comment (Nikhil-Ladha) [reply src/collections/programs/gsoc-2019/index.mdx:14]: Rest all looks good.

**Review by Nikhil-Ladha — APPROVED** [review]
Thank you @ashutosh1401 👍

**Comment by leecalcote** [comment]
Whoo-hoo @ashutosh1401! 🎈

**Pull request merged**